src/DynamoUtilities/Md2Html.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+
+namespace Dynamo.Utilities
+{
+    public class Md2Html
```

Comment:
curious what you think about making this IDisposable

Reply:
I tried IDisposal and yes, much easier to control. I don't need the ProcessExit handler anymore. So I keep the IDisposal implementation for now.